route/build/src/mpi_process.f90

```diff
@@ -1124,6 +1139,36 @@ subroutine mpi_restart(pid,           & ! input: proc id
     end do
   endif ! (onRoute(impulseResponseFunc))
 
+  if (tracer) then
+    ! tracer mass  communication
+    if (masterproc) then
+      do iSeg = 1, nRch
+        vol_global_tmp(iSeg) = RCHFLX(iens,iSeg)%ROUTE(idxDW)%reach_solute_mass(1)
```

Comment:
Another use of a 1 in an index.

Reply:
0 and 1 will be replaced with parameters. This change (issue #532) will be made in another PR.